### DISCUSSION TITLE
Questions on Hardware Acceleration and Coral

### DISCUSSION THREAD

**Discussion by TheeDudenator** [blakeblackshear/frigate#9653] [General]
Never heard of this Google Coral, but I picked up a used USB one cheap.

I think I have it configured correctly, not sure how to tell.

Should I still use hardware acceleration?
AMD system with AMD GPU.

Would I be better with a PCIe type Coral?

I have 8 cameras and maybe 10max in the future.

![Screenshot_20240204_103805_FrigateViewer](https://github.com/blakeblackshear/frigate/assets/154283268/9b21a683-7381-4045-b9a9-de9105e47ec2)

**Comment by NickM-27** [comment]
hardware acceleration is recommended to be used along with a coral or other object detector
